cli/src/cli_util.rs

```diff
@@ -2950,7 +2951,13 @@ impl DiffSelector {
                 // whereas we want to update the left tree. Unmatched paths
                 // shouldn't be based off the right tree.
                 let right_tree = right_tree.store().get_root_tree(&selected_tree_id)?;
-                Ok(editor.edit(left_tree, &right_tree, matcher, format_instructions)?)
+                Ok(editor.edit(
+                    left_tree,
+                    &right_tree,
+                    matcher,
+                    format_instructions,
+                    InitialSelection::None,
```

Comment:
comment (non-blocking): We could consider if we want to bundle the "configuration" kind of arguments here into a single `struct` for organizational reasons, but I don't feel strongly.